userspace/libsinsp/parsers.cpp

```diff
@@ -3019,7 +3019,12 @@ void sinsp_parser::parse_open_openat_creat_exit(sinsp_evt *evt)
 			{
 				name = enter_evt_name;
 				namelen = enter_evt_namelen;
-				flags = enter_evt_flags;
+
+				// keep PPM_O_F_CREATED flag if present
+				if (flags | PPM_O_F_CREATED)
```

Comment:
Shouldn't this be:
```suggestion
				if (flags & PPM_O_F_CREATED)
```

And the same for the below usage?

Reply:
oops, my bad!